languages/elm/maintainers.md

```diff
@@ -4,8 +4,7 @@ These awesome people help maintain the Elm track.
 
 ## Senior Maintainers
 
-TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-Cedd Burge
+- Cedd Burge (Slack: `@Cedd burge`, GitHub: [ceddlyburge](https://github.com/ceddlyburge), GitLab: [ceddlylburge](https://gitlab.com/ceddlyburge))
```

Comment:
Can you capitalise your last name? Even though your Slack username has a lowercase `b`, it will look nicer on the list of maintainers and your Slack username is still pingable with a capitalised `b`.

*Perhaps you want to capitalise your last name is Slack too?*

Reply:
I've pushed a commit to capitalise my last name.